spec/class.dd

```diff
@@ -1317,7 +1317,7 @@ $(D new) $(D $(LPAREN))$(I ArgumentList)$(D $(RPAREN)) $(I Identifier) $(D $(LPA
 
 where $(I Identifier) is the name generated for the anonymous nested class.
 
-$(SECTION3 $(LEGACY_LNAME2 ConstClass, const-class, Const, Immutable and Shared Classes),
+$(SECTION2 $(LEGACY_LNAME2 ConstClass, const-class, Const, Immutable and Shared Classes),
```

Comment:
The doc is weird, only shows "Const". I think you need: `$(SECTION2 $(LEGACY_LNAME2 ConstClass, const-class, $(ARGS Const, Immutable and Shared Classes)),`